src/main/java/io/fabric8/elasticsearch/plugin/kibana/DocumentBuilder.java

```diff
@@ -25,8 +25,8 @@
values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"USERSPACE_USEC.raw\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"INITRD_USEC\",\"indexed\":true,\"analyzed\":true,\"doc_values\":false,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"INITRD_USEC.raw\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.t.MACHINE_ID\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.u.RESULT\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.t.EXE\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"SESSION_ID.raw\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.u.CODE_FUNCTION\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.u.SYSLOG_PID\",\"indexed\":true,\"analyzed\":true,\"doc_values\":false,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.t.CMDLINE\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.t.AUDIT_SESSION\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"rsyslog.structured-data\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.k.KERNEL_SUBSYSTEM\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.k.KERNEL_DEVICE\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.t.SYSTEMD_CGROUP\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.k.UDEV_DEVLINK\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"USER_ID.raw\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"systemd.k.UDEV_DEVNODE\",\"indexed\":true,\"analyzed\":false,\"doc_values\":true,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"_id\",\"indexed\":false,\"analyzed\":false,\"doc_values\":false,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"_type\",\"indexed\":false,\"analyzed\":false,\"doc_values\":false,\"type\":\"string\",\"scripted\":false},{\"count\":0,\"name\":\"_score\",\"indexed\":false,\"analyzed\":false,\"doc_values\":false,\"type\":\"number\",\"scripted\":false}]";
```

Comment:
Maybe more of a follow-on PR, but does it make sense to move this to a file we pull in so that it is:

* easier to change and/or update
* can be formatted and easier to read

@ewolinetz  ??

otherwise LGTM